File: GoLang/Day2.go
package main

import "fmt"

func printArray(numbers [5]int){
	for index,_:=range numbers{
		fmt.Print(numbers[index]," ")
	}
	fmt.Println()
}

func changeArray(numbers * [5]int){
	for i,_:=range(numbers){
		numbers[i]=88;
	}
}

func playwithArraysPointers(){
var numbers [5]int=[5]int {10,20,30,40,50};

printArray(numbers);
changeArray(&numbers);
printArray(numbers);
}

func main(){
playwithArraysPointers();
}

File: GoLang/Day2.java
class PlayWithArrayPointers{
    public  void printArray(int[] arr){
        for(int num:arr){
            System.out.print(num+" ");
        }
        System.out.println(" ");
    }

    public void changeArray(int[] arr){
        for(int i=0;i<arr.length;i++){
            arr[i]=777;
        }
    }  
}
public class Day2 {
    public static void main(String[] args) {
        PlayWithArrayPointers ply=new PlayWithArrayPointers();
        int[] numbers=new int[]{10,20,30,40,50};
        ply.printArray(numbers);
        ply.changeArray(numbers);
        ply.printArray(numbers);
    }
}
